### ISSUE TITLE
Authorization: Bearer [object Object] 4.7.0

### ISSUE THREAD

**Issue by READYTOMASSACRE** [swagger-api/swagger-ui#7926]
- OS: windows
 - Browser: chrome
 - Version: 99.0.4844.74
 - Method of installation: unpkg.com/swagger-ui-dist@latest
 - Swagger-UI version: 4.7.0

1. Go to main page
2. Click on Authorize button to open modal
![image](https://user-images.githubusercontent.com/7259842/159471533-8657c726-bc2e-42ca-aebd-1dd68fba4a6f.png)
3. Insert jwt token 
4. In headers it will be [object Object] instead of token 
![image](https://user-images.githubusercontent.com/7259842/159472116-3606a993-8e2d-4bd4-9303-4cbe8d5b6064.png)

**Comment by guicompeng** [comment]
same in 4.8.0

**Comment by hkosova** [comment]
This seems to be a regression in v. 4.7.0.

Sample spec to reproduce the issue:
```yaml
openapi: 3.0.0
info:
  title: Bearer auth test
  version: 1.0.0
servers:
  - url: https://httpbin.org
paths:
  /get:
    get:
      responses:
        '200':
          description: ok
security:
  - bearerAuth: []
components:
  securitySchemes:
    bearerAuth:
      type: http
      scheme: bearer
```

**Comment by guillermo-mendez** [comment]
in nodeJS with express ("swagger-ui-express": "^4.1.6",) fixes the problem

**Comment by tim-lai** [comment]
Thanks all for the issue report and confirmation! Fixed in [v4.8.1](https://github.com/swagger-api/swagger-ui/releases/tag/v4.8.1)
